plugin-server/src/main/ingestion-queues/batch-processing/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { EachBatchPayload, KafkaMessage } from 'kafkajs'
```

Comment:
Note I hate utils.ts files since by definition they can contain anything - Can we give this a better name? E.g. `eachBatch.ts`?